ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/inband-mgmt-spine2-ztp.cfg

```diff
@@ -7,7 +7,6 @@ daemon TerminAttr
    no shutdown
 !
 dhcp server
-   dns server ipv4 1.1.1.1 8.8.8.8
```

Comment:
TODO check this is still covered somewhere

Reply:
I missed this. I will fix